inference-engine/tests_deprecated/fluid_preproc/fluid_test_computations/fluid_test_computations.cpp

```diff
@@ -12,8 +12,42 @@
 struct FluidComputation::Priv
 {
     cv::GComputation m_c;
-    std::vector<cv::gapi::own::Mat> m_v_in;
+    cv::GRunArgs m_v_in;
     std::vector<cv::gapi::own::Mat> m_v_out;
+#if 1
+    Priv(cv::GComputation && c, std::vector<cv::gapi::own::Mat>&& v_in, std::vector<cv::gapi::own::Mat>&& v_out)
+        : m_c(std::move(c)),
+          m_v_in(v_in.begin(), v_in.end()),
+          m_v_out(std::move(v_out))
+    {}
+
+    Priv(cv::GComputation && c, cv::gapi::own::Mat&& v_in, cv::gapi::own::Mat&& v_out)
+        : m_c(std::move(c)),
+          m_v_in{std::move(v_in)},
+          m_v_out{std::move(v_out)}
+    {}
+
+    Priv(cv::GComputation && c, cv::gapi::own::Mat&& v_in, std::vector<cv::gapi::own::Mat>&& v_out)
```

Comment:
I could be wrong but `std::initializer_list<>` can reduce multiple constructor definistions

     Priv(..., std::initializer_list<own::Mat> v_in, std::initializer_list<own::Mat> v_out) :
      ...
       m_v_in(v_in),
       m_v_out(v_out)
     { }

Reply:
Each constructors has different types of arguments. So in my opinion, std::initializer_list<> don't fit for this case.